hubmap/portal/tests.py

```diff
@@ -0,0 +1,37 @@
+from django.test import RequestFactory, TestCase
+
+# Create your tests here.
+from .views import details, entities, index, listall
+
+
+class PortalViewsTestCase(TestCase):
+    def setUp(self) -> None:
+        self.factory = RequestFactory()
+        self.request = self.factory.request()
+        self.entity_type = 'donor'
+        self.hubmap_id = 4
+        pass
+
+    def test_home_page(self):
+        response = index(self.request)
+        self.assertEqual(response.status_code, 200)
+        self.assertIsNotNone(response.content)
+        pass
+
+    def test_list_entities_page(self):
+        response = listall(self.request)
+        self.assertEqual(response.status_code, 200)
+        self.assertIsNotNone(response.content)
+        pass
```

Comment:
Remove extra `pass` (also two more later in this file)

Reply:
fixed